src/main/java/no/finn/unleash/util/UnleashConfig.java

```diff
@@ -88,8 +101,17 @@ public UnleashContextProvider getContextProvider() {
         return contextProvider;
     }
 
+    public static void setRequestProperties(HttpURLConnection connection, UnleashConfig config) {
+        connection.setRequestProperty(UNLEASH_APP_NAME_HEADER, config.getAppName());
+        connection.setRequestProperty(UNLEASH_INSTANCE_ID_HEADER, config.getInstanceId());
+        for (String name : config.getCustomHttpHeaders().keySet()) {
```

Comment:
Is this module java 8? If so, can use `config.getCustomHttpHeaders().keySet().forEach`. If _not_ java 8, ignore!